### DISCUSSION TITLE
ElastoDyn input files

### DISCUSSION THREAD

**Discussion by KarimMohammadd** [OpenFAST/openfast#1129] [Q&A]
Hello everyone
We know that one of the ElastoDyn input files is a file containing tower properties. I wonder if there is any user guide available about how to make this tower properties data file.
Do you have any idea please?
Thank you in advance!

**Comment by jjonkman** [comment]
Dear @KarimMohammadd,

At present, the ElastoDyn documentation is limited in the online OpenFAST readthedocs: https://openfast.readthedocs.io/en/main/source/user/elastodyn/index.html.  However, the old documentation from FAST v7 well described most of the structural-related inputs that were migrated from FAST v7 into the ElastoDyn module.  You can find that here: https://openfast.readthedocs.io/en/main/_downloads/d8bd014121d6505cb25cf49bee5eaa80/Old_FAST6_UsersGuide.pdf.  Structural updates made during the conversion from FAST v7 to the modularized framework used by FAST v8 and OpenFAST are documented here: https://openfast.readthedocs.io/en/main/_downloads/5f2ddf006568adc9b88d8118dc3f1732/FAST8_README.pdf.

Best regards,

**Reply by KarimMohammadd** [reply]
Thank you so much MR Jonkman.
